packages/components/src/components/domainproperties/constants.ts

```diff
@@ -193,3 +194,5 @@ export const FIELD_NAME_CHAR_WARNING_INFO = 'SQL queries, R scripts, and other c
     + 'names contain only a combination of letters, numbers, and underscores, and start with a letter or underscore. '
     + 'Exporting fields that abide by this format are also easier to work with in analysis applications such as SAS, '
     + 'SPSS, and MATLAB.';
+
+export const KINDS = Domain.KINDS;
```

Comment:
Any reason not to just skip this extra const and just use Domain.KINDS directly within the various places in this repo?

Reply:
Was using this in SM, instead of adding a new dependency on api-js

Reply:
SM should already have access to use @labkey/api since it is pulled in because of the @labkey/components dependency, so I believe you should already be able to use Domain.KINDS now that your @labkey/components alpha package has your updated @labkey/api version.